week01 - damdam6 - cs-study

damdam6 opened the pull request:
- MongoDB 자세히 알아보기
- sharding
- replica set

damdam6 committed:
feat(week01): mongoDB자세히보기-초안

damdam6 committed:
fix: 파일 오류 수정

Zerotay reviewed (commented):
샤딩을 할 경우 데이터 정합성을 어떻게 유지할 수 있는지 궁금합니다

Damongsanga commented:
보통 SpringBoot랑 Mysql을 같이 쓰고, Node.js랑 mongoDb 조합을 많이 쓰는데 왜그런지 궁금합니다. 그리고 mongodb를 사용하면서 대표적으로 주의해야 할 점에 대해 알고 싶어요

bbookng commented:
NoSQL에 대해서 및 다른 NoSQL 과의 간단한 비교 내용, 그리고 MongoDB를 어떨 때 사용하면 좋을 지 내용이 추가되면 좋을 것 같습니다 !

wonjunJ commented:
보통 NoSQL하면 떠올리는 Redis와 MongoDB의 차이에 대해서 설명해 주시면 좋을 것 같습니다. 언제 어느경우에 MongoDB를 쓰는것이 좋은지도 설명해 주시면 좋을 것 같아요!

damdam6 committed:
feat(week01): MongoDB 자세히 알아보기

- 설명용 pdf 추가
- 일부 설명 추가 및 순서 수정

- 주제 및 흐름
-- Nosql 간단 설명
-- Redis vs MongoDB
-- MongoDB - Sharding
-- MongoDB - Replica Sets
